api/sql/schema/750-rule.decision.fetch.sql

```diff
@@ -358,7 +358,7 @@ BEGIN TRY
         @rateName rateConditionName,
         CONVERT(VARCHAR(21), ROUND((SELECT SUM(ISNULL(fee, 0)) FROM @fee WHERE tag LIKE '%|acquirer|%' AND tag LIKE '%|fee|%'), @scale), 2) acquirerFee,
         CONVERT(VARCHAR(21), ROUND((SELECT SUM(ISNULL(fee, 0)) FROM @fee WHERE tag LIKE '%|issuer|%' AND tag LIKE '%|fee|%'), @scale), 2) issuerFee,
-        CONVERT(VARCHAR(21), ROUND((SELECT SUM(ISNULL(fee, 0)) FROM @fee WHERE tag LIKE '%|processor|%' AND tag LIKE '%|fee|%'), @scale), 2) processorFee,
+        CONVERT(VARCHAR(21), ROUND((SELECT SUM(ISNULL(fee, 0)) FROM @fee WHERE (tag LIKE '%|processor|%' AND tag LIKE '%|fee|%') OR (tag LIKE '%|fee|%' AND tag NOT LIKE '%|acquirer|%' AND tag NOT LIKE '%|issuer|%')), @scale), 2) processorFee,
```

Comment:
We must not change the existing logic here. The processor fee is calculated from the rows tagged with `processor` and `fee` tags, so the code is correct.

Reply:
@tuncheto